sdk/keyvault/Azure.Security.KeyVault.Keys/tests/Azure.Security.KeyVault.Keys.Tests.csproj

```diff
@@ -19,4 +19,8 @@
     <ProjectReference Include="..\..\..\identity\Azure.Identity\src\Azure.Identity.csproj" />
     <ProjectReference Include="..\src\Azure.Security.KeyVault.Keys.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <None Update="TestData\**\*" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Why did you have to do this?

Reply:
Yeah I suppose I could have just listed the folder, but I copied it from the session records and edited.  But yes the wildcards are superfluous